package.json

```diff
@@ -6,24 +6,22 @@
     "packages/*"
   ],
   "dependencies": {
-    "husky": "8.0.1",
-    "lerna": "5.5.0"
+    "husky": "8.0.1"
   },
   "devDependencies": {
     "lerna-update-wizard": "1.1.0"
   },
   "scripts": {
-    "build": "lerna run build --ignore datagateway-common",
-    "lint": "lerna run lint:js -- --max-warnings=0",
-    "tsc": "lerna run tsc",
+    "build": "yarn workspaces foreach --verbose --parallel --exclude datagateway-common run build",
```

Comment:
- Can we add the `--interlaced` flag to this so we can see the output?
- If we add `--jobs 3` to this, I think it might be slightly faster (it seemed that way on my machine)